src/frontends/paddle/src/frontend.cpp

```diff
@@ -135,22 +135,32 @@ bool normalize_framework_node(const std::shared_ptr<FrameworkNode>& node,
     return true;
 }
 
-std::istream* variant_to_stream_ptr(const ov::Any& variant, std::ifstream& ext_stream) {
+OPENVINO_SUPPRESS_DEPRECATED_START
```

Comment:
It seems `ngraph::runtime::AlignedBuffer` will be deprecated, but this function will not, right?

Reply:
> It seems `ngraph::runtime::AlignedBuffer` will be deprecated, but this function will not, right?

@riverlijunjie Yes, your understanding is correct. The function won't be deprecated.

Reply:
@ilyachur @xczhai Is it okay to declare OPENVINO_SUPPRESS_DEPRECATED_START and OPENVINO_SUPPRESS_DEPRECATED_END only for the branch of AlignedBuffer instead of the whole fucntion?